test/common.py

```diff
@@ -380,6 +380,7 @@ def metafunc(self, wasmfs, *args, **kwargs):
       print('parameterize:wasmfs=%d' % wasmfs)
     if wasmfs:
       self.set_setting('WASMFS')
+      self.set_setting('FORCE_FILESYSTEM')
```

Comment:
Why is this needed?

This PR only affects the one test `test_fs_writeFile`, which already has the decorator, as you mention in the description. Adding this flag here seems unnecessary IIANM, and has the risk of avoiding us from ever testing the non-forced case with wasmfs.